apps/chat/src/components/Chat/ModelList.tsx

```diff
@@ -118,9 +115,7 @@ const ModelGroup = ({
 
   const description = currentEntity.description;
   const currentEntityId = currentEntity.id;
-  const isPublishedEntity = publishedApplicationIds.some(
-    (id) => id === currentEntityId,
-  );
+  const isPublishedEntity = currentEntityId.includes('public');
```

Comment:
That's absolutely wrong.
What if I have public just in name or somewhere else?
please, use isRootId method, and as bucket prop pass PUBLIC_URL_PREFIX